scripts/host_cap.py

```diff
@@ -329,7 +334,11 @@ def add_constituent_vars(cap, host_model, suite_list, run_env):
                 if len(dims) > 1:
                     vdim = dims[1].split(':')[-1]
                     if vdim == vert_layer_dim:
-                        cvar_array_name = array_layer
+                        if is_tend_var:
+                            cvar_array_name = tend_layer
+                        else:
+                            cvar_array_name = array_layer
```

Comment:
What's the difference between tend_layer and array_layer?

Reply:
Or, what does the `layer` in these names stand for? Might be showing my ignorance here.

Reply:
a great question. `layer` comes from the fact that both the constituent data array (array_layer=vars_layer) and the tendency array (tend_layer=vars_layer_tend) use the `vertical_layer_dimension` dimension (rather than interface). 

All that said, I'm open to any suggestions to clarify the naming conventions!

Reply:
All good, thanks for the clarification